CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
   [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
   [Semantic Versioning]: http://semver.org/spec/v2.0.0.html
 
+## [0.6.1] - 2023-09-29
```

Comment:
We tend to do this on a different PR - as we also have to bump in `setup.py`

Reply:
Ah ok, will remove 👍 I did also update setup.py to 0.6.1, should I also undo that change?

Reply:
Yup.